qiskit_ibm_runtime/utils/estimator_pub_result.py

```diff
@@ -0,0 +1,83 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+========================================================================================
+EstimatorPubResult result classes (:mod:`qiskit_ibm_runtime.utils.estimator_pub_result`)
+========================================================================================
+
+.. autosummary::
+   :toctree: ../stubs/
+"""
+
+from __future__ import annotations
+
+from typing import Sequence
+import plotly.graph_objects as go
+
+from qiskit.primitives.containers import PubResult
+
+
+class EstimatorPubResult(PubResult):
+    """Result of Estimator Pub."""
+
+    def plot_zne(
+        self,
+        indices: Sequence[tuple[int, ...]] | None = None,
+        names: Sequence[str] | None = None,
+        n_stds: int = 1,
+        mag_tol: float = 10,
+        std_tol: float = 0.2,
+        height: int = 500,
+        width: int = 1000,
+        n_cols: int = 4,
+        colorscale: str = "Aggrnyl",
+        subplots: bool = False,
+    ) -> go.Figure:
+        """Plot the zero noise extrapolation data contained in this estimator pub result.
+
+        Args:
+            indices: The indices of the expectation values to include in the plot. If ``None``, includes
+                all values. See :class:`~.ZneOptions` for information on the indexing scheme.
+            names: The names to assign to the expectation values. If ``None``, the names correspond to
+                the indices.
+            n_stds: The number of standard deviations to include around each fit.
+            mag_tol: The tolerance. If ``evs_extrapolated`` has a greater magnitude than this value, the
```

Comment:
Consider using some other name (maybe `max_evals` and `stds_max`?), since the concept of magnitude of an evs doesn't show up anywhere else in qiskit-ibm-runtime (that I know of)